docs/guide/images.md

```diff
@@ -225,3 +225,127 @@ The following filters exist in the Imagine library but are not used in eZ Platfo
+- [RemoteProvider](#remoteprovider)
+
+```php
+<?php
+
+namespace eZ\Bundle\EzPublishCoreBundle\Imagine;
+
+use eZ\Publish\Core\FieldType\Image\Value as ImageValue;
+
+interface PlaceholderProvider
+{
+    /**
+     * Provides a placeholder image path for a given Image FieldType value.
+     *
+     * @param \eZ\Publish\Core\FieldType\Image\Value $value
+     * @param array $options
+     * @return string Path to placeholder
+     */
+    public function getPlaceholder(ImageValue $value, array $options = []): string;
+}
+```
+
+### GenericProvider
+
+`\eZ\Bundle\EzPublishCoreBundle\Imagine\PlaceholderProvider\GenericProvider` generates placeholder with basic information about original image - [example 1](#configuration-examples). 
+
+**Generic image example:**
+
+![Placeholder image GenericProvider](img/placeholder_info.jpg)
+
+**Full page example:**
+
+![Placeholder GenericProvider](img/placeholder_generic_provider.png)
+
+|Option|Default value|Description|
+|------|-------------|-----------|
+|fontpath|n/a|Path to the font file (*.ttf). **This option is required.**|
+|text|"IMAGE PLACEHOLDER %width%x%height%\n(%id%)"|Text which will be displayed in the image placeholder. %width%, %height%, %id% in it will be replaced with width, height and ID of the original image.|
+|fontsize|20|Size of the font in the image placeholder.|
+|foreground|#000000|Foreground color of the placeholder.|
+|secondary|#CCCCCC|Secondary color of the placeholder.|
+|background|#EEEEEE|Background color of the placeholder.|
+
+### RemoteProvider
+
+`\eZ\Bundle\EzPublishCoreBundle\Imagine\PlaceholderProvider\RemoteProvider` allows you to download placeholders from:
+ 
+ - remote source e.g. <http://placekitten.com> - [example 2](#configuration-examples)
+ - live version of a site - [example 3](#configuration-examples)
+
+**Full page example:**
+
+![Placeholder RemoteProvider - placekitten.com](img/placeholder_remote_provider.jpg)
+
+|Option|Default value|Description|
+|------|-------------|-----------|
+|url_pattern|''|URL pattern. %width%, %height%, %id% in it will be replaced with width, height and ID of the original image.|
+|timeout|5|Period of time before timeout, measured in seconds.|
+
+### Semantic configuration
+
+Placeholder generation can be configured for each [`binary_handler`](/file_management/#handling-binary-files) under the `ezpublish.image_placeholder` key:
+
+```yaml
+ezpublish:
+    # ...
+    image_placeholder:
+        <BINARY_HANDLER_NAME>:
+            provider: <PROVIDER TYPE>
+            options:  <CONFIGURATION>
+```
+
+If there is no configuration assigned to [`binary_handler`](/file_management/#handling-binary-files) name, the placeholder generation will be disabled.
+
+##### Configuration examples:
+
+**Example 1 - placeholders with basic information about original image**
+
+```yaml
+ezpublish:
+    # ...
+    image_placeholder:
+        default:
+            provider: generic
+            options:
+                fontpath:   ''
```

Comment:
I'm thinking maybe it would be nice to provide an example for font. I spent some time trying to set this up. Of course this is optional feedback. \cc @adamwojs